cpu.tf

```diff
@@ -0,0 +1,33 @@
+resource "datadog_monitor" "cpu_average" {
+  name    = "${module.label.id}"
+  type    = "${var.alert_type}"
+  message = "High CPU usage last ${var.cpu_time} on host: ${data.aws_instance.monitored.instance_id} with IP: ${data.aws_instance.monitored.instance_id.public_ip}"
+  query   = "avg(last_${var.cpu_time}):avg:system.cpu.system{host:${data.aws_instance.monitored.instance_id}} by {host} > ${var.cpu_critical_state_value}"
```

Comment:
I don't like how rigid this solution is. How would we monitor memory? I think maybe this module should be renamed to `tf_datadog_host_cpu` **or** we define a `null_resource` with multiple templates like `host_cpu` and the user passes in a parameter which indicates which template to use. That way we can define lots of preset templates for monitoring. @comeanother  thoughts?